AppCenterCrashes/AppCenterCrashes/Model/MSACExceptionModel.h

```diff
@@ -67,7 +67,7 @@ NS_SWIFT_NAME(ExceptionModel)
 /**
  * Stack frames [optional].
  */
-@property(nonatomic) NSArray<MSACStackFrame *> *frames;
+@property(nonatomic, copy) NSArray<MSACStackFrame *> *frames;
```

Comment:
It was `strong` before (as a default attribute). Any reason to change it to `copy`?
I would prefer using `strong` on `NSArray`.